Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla38105.cs

```diff
@@ -1,22 +1,15 @@
-﻿using System;
-
-using Xamarin.Forms.CustomAttributes;
+﻿using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
-#if UITEST
-using Xamarin.UITest;
-using NUnit.Framework;
-#endif
 
-namespace Xamarin.Forms.Controls
+namespace Xamarin.Forms.Controls.Issues
 {
 	[Preserve (AllMembers = true)]
-	[Issue (IssueTracker.Bugzilla, 38105, "RemovePage does not cause back arrow to go away on Android",
-		NavigationBehavior.PushModalAsync)]
-	internal class Bugzilla38105 : TestMasterDetailPage
+	[Issue (IssueTracker.Bugzilla, 38105, "RemovePage does not cause back arrow to go away on Android", PlatformAffected.Android, navigationBehavior: NavigationBehavior.PushModalAsync)]
+	public class Bugzilla38105 : TestMasterDetailPage
```

Comment:
I don't understand the use case for the `NavigationBehavior` enum and also think the given value for this test might be incorrect.

Reply:
it means when you go to this issue it gets pushed as a modal because the test env requires it.